sentry-core/src/main/java/io/sentry/core/exception/SentryFrame.java

```diff
@@ -0,0 +1,86 @@
+package io.sentry.core.exception;
+
+import java.lang.reflect.Method;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+// TODO: should we use that? why?
+public class SentryFrame {
+  /**
+   * Method that this frame originated in.
+   */
+  private Method method;
+  /**
+   * Local variable information for this frame.
+   */
+  private final LocalVariable[] locals;
```

Comment:
This we don't need. It exists to support Sentry Java Agent:
https://github.com/getsentry/sentry-java/issues/617